web/src/server/api/routers/sessions.ts

```diff
@@ -285,6 +285,18 @@ export const sessionRouter = createTRPCRouter({
           }),
           // costData
           ctx.prisma.$queryRaw<Array<{ totalCost: number }>>(totalCostQuery),
+          // traceCommentCounts
+          ctx.prisma.comment.groupBy({
+            by: ["objectId"],
+            where: {
+              objectId: { in: session.traces.map((t) => t.id) },
+              projectId: input.projectId,
+              objectType: "TRACE",
+            },
+            _count: {
+              objectId: true,
+            },
+          }),
```

Comment:
I'm concerned about added latency, as this query is expensive. I'll add a DD metric prior to merging this code